doc/source/ray-air/doc_code/tf_starter.py

```diff
@@ -66,6 +59,9 @@ def train_func(config: dict):
 
 config = {"lr": 1e-3, "batch_size": 32, "epochs": 4}
 
+items = [i / size for i in range(size)]
+dataset = ray.data.from_items([{"x": x, "y": a * x + b} for x in items])
```

Comment:
Thanks @Yard1!

Can we keep the dataset the same as in torch quick start?

```
train_dataset = ray.data.from_items([{"x": x, "y": 2 * x + 1} for x in range(200)])
```

Reply:
it had to be normalized to be between 0 and 1 to make TF work correctly.